files/fancy-scroll/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "fancy-scroll",
+  "repo": "peachananr/fancy-scroll",
+  "files": {
+    "include": ["jquery.fancy-scroll.js", "jquery.fancy-scroll.min.js"]
```

Comment:
There are no tags here https://github.com/peachananr/fancy-scroll/releases
The bot will fail